deploy/helm/events-collector-config.yaml

```diff
@@ -116,6 +116,17 @@ processors:
         statements:
           # convert body to string
           - set(body, body.string)
+  transform/set_status_for_pods:
+    error_mode: ignore
+    log_statements:
+      - context: log
+        statements:
+          # status
+          - set(attributes["sw.k8s.pod.status"], body["status"]["phase"]) where body["kind"] == "Pod" and body["status"]["phase"] != nil
```

Comment:
Is the phase value here identical to the value passed in the metric kube_pod_status_phase? No change in letter casing etc.?